unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs

```diff
@@ -84,6 +83,10 @@ private void BlitTexture()
 
         private void SetupBlitTexture()
         {
+            //Blit null works differently in WebGL than in desktop platform. We have to deal with it and rotate the resultant image accordingly
+            if (Application.isEditor || Application.platform != RuntimePlatform.WebGLPlayer)
+                rawImage.GetComponent<RectTransform>().eulerAngles = new Vector3(180, 0, 0);
```

Comment:
Can you cache the `rawImage.GetComponent<RectTransform>()` call?

Reply:
Done